ReactAndroid/src/main/java/com/facebook/react/modules/accessibilityinfo/AccessibilityInfoModule.java

```diff
@@ -58,6 +58,7 @@ public void onChange(boolean selfChange) {
         public void onChange(boolean selfChange, Uri uri) {
           if (getReactApplicationContext().hasActiveCatalystInstance()) {
             AccessibilityInfoModule.this.updateAndSendReduceMotionChangeEvent();
+            AccessibilityInfoModule.this.updateAndSendInvertColorsChangeEvent();
```

Comment:
This isn't going to work as part of the same observer.

The way these observers work is that the URI passed into them is a reference to the specific setting changed, and this one is being trigged on `Settings.Global.getUriFor(Settings.Global.TRANSITION_ANIMATION_SCALE);` (line 175 of this file), which is the URI for the animation_scale setting, *not* the display inversion setting.

You'll instead either need to make a second ContentObserver, or use the same observer as you have done, but add a check for the URI in the onChange method and only fire the update method associated with the URI that has triggered it. Whichever approach you opt for you'll also still need to call registerContentObserver with your new URI on line 176 like this:

`mContentResolver.registerContentObserver(Settings.Secure.getUriFor(Settings.Secure.ACCESSIBILITY_DISPLAY_INVERSION_ENABLED), false, YOUR_CONTENT_RESOLVER_HERE);`

Let me know if any of this needs any clarification!